pyot/aggregate.py

```diff
@@ -0,0 +1,22 @@
+from pyspark.sql.functions import col, from_json, window
+from pyot import schema
+
+
+def create_averaged_df(raw_streaming_df, window_sec, watermark_sec):
```

Comment:
Would be good to add type hints to your functions, ei `raw_streaming_df: DataFrame`. Also, a docstring would be nice.

Reply:
Thank you for your suggestion. Added type hints and docstrings here.